tests/test_item.py

```diff
@@ -171,6 +174,34 @@ class E(C, B): pass
         self.assertEqual(E.fields, {'load': {'default': 'C'},
             'save': {'default': 'C'}})
 
+        class TestItem(Item):
+            name = Field()
+
+        class ChildItem(TestItem):
+            name = 'John'
+            age = Field()
+
+        item = ChildItem()
+        self.assertEqual(item.get('name'), 'John')
+        self.assertRaises(KeyError, item.__getitem__, 'age')
+
+        class GrandChildItem(ChildItem):
+            age = 20
```

Comment:
I think there might be a misunderstanding here. Of course items can have arbitrary attributes, properties and methods just like any other class, but for an attribute to be treated like a _field_ it should be declared as a `scrapy.item.Field` object (https://docs.scrapy.org/en/1.8/topics/items.html#declaring-items). Otherwise, unexpected behaviour can occur (for instance, some components like [item exporters](https://github.com/scrapy/scrapy/blob/1.8/scrapy/exporters.py#L64) check the item's `fields` attribute).